README.md

```diff
@@ -23,10 +23,38 @@ The arguments field on the New Action panel should be given the following:
 
 Also, the polling interval should match the value used in the "Repeat task every" field in the New Trigger dialog.  
 
+# Example Installation 
 Click the link (and then 'View Raw')for a video configuration walkthrough of Signal iQ on Windows Server 2016:
 
 [Signal iQ Configuration](../master/Signal_iQ.webm)
 
 Click the link (and then 'View Raw')for a video configuration walkthrough of Windows Signal:
 
 [Windows Signal Configuration](../master/Windows_Signal.webm)
+
+The following PowerShell code will configure a Task after prompting for all the required inputs:
+
+```
+# prompt user for important paths
```

Comment:
Question. Why not create a file that captures that? This way it can be versioned, etc. as the rest of the code.

Reply:
We can do that if you'd rather.